includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -620,11 +656,41 @@ static function() use ( $handles ) {
 	 * @param array $actions Actions, with action name as key and value being callback.
 	 */
 	public static function remove_actions( $actions = [] ) {
+		global $wp_filter;
+
 		foreach ( $actions as $action => $callbacks ) {
 			foreach ( $callbacks as $callback ) {
 				$priority = has_action( $action, $callback );
```

Comment:
This check could be avoided if `$callback` not `is_callable()`, such as the case where `$callback` is an array of 3 items, yeah?

Reply:
Would something like this suffice?

```php
if ( is_callable( $callback ) ) {
	$priority = has_action( $action, $callback );
	if ( false !== $priority ) {
		remove_action( $action, $callback, $priority );
		continue;
	}
}
```